### PR TITLE
ovirt-external-providers: modify documentation

### PR DESCRIPTION
**Pull request by tareqalayan**
add documentation regarding removing a provider.

### CODE REVIEW

File: roles/ovirt-external-providers/README.md [hunk 1]
```diff
@@ -21,13 +21,13 @@ The items in `external_providers` list can contain the following parameters:
 | Name                   | Default value       | Description                                                                      |
 |------------------------|---------------------|----------------------------------------------------------------------------------|
 | name                   | UNDEF (Required)    | Name of the external provider.                                                   |
-| state                  | present             | State of the external provider.                                                  |
+| state                  | present             | State of the external provider. Values can be: <ul><li>present</li><li>absent</li></ul>|
 | type                   | UNDEF (Required)    | Type of the external provider. Values can be: <ul><li>os_image</li><li>network</li><li>os_volume</li><li>foreman</li></ul>|
-| url                    | UNDEF (Required)    | URL where external provider is hosted.                                          |
+| url                    | UNDEF (Required)    | URL where external provider is hosted. Not required if state is absent          |
```
Comment (machacekondra) [inline roles/ovirt-external-providers/README.md]: I think better to remove the `(Requrired)` from default value. and in description say Required if state is present.

File: roles/ovirt-external-providers/README.md [hunk 2]
```diff
@@ -21,13 +21,13 @@ The items in `external_providers` list can contain the following parameters:
...
 | username               | UNDEF               | Username to be used for login to external provider. Applicable for all types.   |
 | password               | UNDEF               | Password of the user specified in username parameter. Applicable for all types. |
 | tenant                 | UNDEF               | Name of the tenant. |
-| auth_url               | UNDEF               | Keystone authentication URL of the openstack provider. required: <ul><li>os_image</li><li>network</li><li>os_volume</li></ul>|
+| auth_url               | UNDEF               | Keystone authentication URL of the openstack provider. Required: <ul><li>os_image</li><li>network</li><li>os_volume</li></ul>|
```
Comment (machacekondra) [inline roles/ovirt-external-providers/README.md]: `Required for:` would be better imho

File: roles/ovirt-external-providers/README.md (hunk 1 above)
Comment (tareqalayan) [reply roles/ovirt-external-providers/README.md]: fixed

File: roles/ovirt-external-providers/README.md (hunk 2 above)
Comment (tareqalayan) [reply roles/ovirt-external-providers/README.md]: fixed.

**Review by machacekondra — APPROVED** [review]

**Pull request merged**